R/utilities.R

```diff
@@ -3115,29 +3117,34 @@ get_bams = function(this_sample_id,
   }else{
     igv_build = build
   }
-  tumour_genome_bams = dplyr::filter(meta_patient, seq_type == "genome" & tissue_status == "tumour") %>%
-    dplyr::pull(data_path)
+  bam_path_pattern = "/projects/rmorin/projects/gambl-repos/gambl-rmorin/data/{seq_type}_bams/{sample_id}.{genome_build}.bam"
```

Comment:
Should we have this pulled from the config?  Like `config::get("repo_path")` and paste with the data/... to generate the string with path, so it is consistent with the rest of the functions?